mopidy_mpd/protocol/music_db.py

```diff
@@ -84,6 +86,62 @@ def _artist_as_track(artist):
     )
 
 
+def _get_art(context, uri=None, offset=0):
+    # TODO work out how validators work and move these there
+    if uri is None:
+        raise exceptions.MpdArgError("Need to specify uri")
+    offset = protocol.INT(offset)
+
+    images = context.core.library.get_images([uri]).get()[uri]
+
+    if len(images) == 0:
+        raise exceptions.MpdNoExistError("No file exists")
+
+    image_uri = images[0].uri
+
+    if image_uri == context.art_cache[0]:
+        bytes = context.art_cache[1]
+    else:
+        if image_uri.startswith("/"):
+            data_path = context.config["core"]["config_dir"]
+            _, extension, file = image_uri.split("/")
+            with open(
+                os.path.join(data_path, extension, "images", file), "rb"
+            ) as image_file:
+                bytes = image_file.read()
+        elif image_uri.startswith("https://"):
+            with urlopen(Request(image_uri)) as r:
```

Comment:
should try-catch-wrap that. example: `urlopen(Request("https://expired.badssl.com/"))`. according to my way-too-late-reading of the documentation, it should raise a subclass of `urllib.error`.